packages/devtools_app/lib/src/timeline/flutter/timeline_service.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:vm_service/vm_service.dart' as vm_service;
```

Comment:
can you provide the baseline version of the code this is based on?

Reply:
see latest commit

Reply:
thanks!